src/Files.App/Converters/Converters.cs

```diff
@@ -238,4 +238,32 @@ protected override Visibility Convert(object? value, object? parameter, string?
 			return new NotSupportedException();
 		}
 	}
+
+	internal sealed partial class EmptyListToVisibilityConverter : ValueConverter<object?, Visibility>
+	{
+		/// <summary>
+		/// Converts a source value to the target type.
+		/// </summary>
+		/// <param name="value"></param>
+		/// <param name="parameter"></param>
+		/// <param name="language"></param>
+		/// <returns></returns>
+		protected override Visibility Convert(object? value, object? parameter, string? language)
```

Comment:
Instead of adding empty comments, simple `<inheritdoc/>` could be used (note that I didn't specify "cref" parameter)